### PR TITLE
Native YAML - monitoring/boundary_meter

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
monitoring/boundary_meter

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

@gundalow

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @Fale. To the current maintainers, @ccollicutt please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: monitoring/boundary_meter.py [hunk 1]
```diff
@@ -68,11 +68,18 @@
 
 EXAMPLES='''
 - name: Create meter
-  boundary_meter: apiid=AAAAAA api_key=BBBBBB state=present name={{ inventory_hostname }}"
+  boundary_meter:
+    apiid: AAAAAA
+    api_key: BBBBBB
```
Comment (gundalow) [inline monitoring/boundary_meter.py]: Original example is wrong, it's `apikey`

File: monitoring/boundary_meter.py [hunk 2]
```diff
@@ -68,11 +68,18 @@
...
+    state: present
+    name: '{{ inventory_hostname }}'
 
 - name: Delete meter
-  boundary_meter: apiid=AAAAAA api_key=BBBBBB state=absent name={{ inventory_hostname }}"
-
+  boundary_meter:
+    apiid: AAAAAA
+    api_key: BBBBBB
```
Comment (gundalow) [inline monitoring/boundary_meter.py]: `apikey`

**Review by gundalow — CHANGES_REQUESTED** [review]
needs_revision 

Not your bug, however I noticed that the use of `_` was inconsistent in the examples.

**Comment by Fale** [comment]
Nice catch!

**Pull request closed without merging**